common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1463,21 +1488,41 @@ object CelebornConf extends Logging {
       .checkValue(Seq("roundrobin", "loadaware").contains(_), "")
       .createWithDefault("roundrobin")
 
-  def initialPartitionSize(conf: CelebornConf): Long = {
-    Utils.byteStringAsBytes(conf.get("rss.initial.partition.size", "64m"))
-  }
+  val INITIAL_ESTIMATED_PARTITION_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.initialEstimatedPartitionSize")
+      .withAlternative("rss.initial.partition.size")
+      .categories("master")
+      .doc("Initial estimated partition size, it will change according to runtime stats.")
+      .version("0.2.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("64mb")
 
-  def minimumPartitionSizeForEstimation(conf: CelebornConf): Long = {
-    Utils.byteStringAsBytes(conf.get("rss.minimum.estimate.partition.size", "8m"))
-  }
+  val MINIMUM_ESTIMATED_PARTITION_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.minEstimatedPartitionSize")
```

Comment:
How about "celeborn.worker.minPartitionSizeForEstimation"? I think this name can match the doc.

Reply:
MinEstimatedPartitionSize looks like that this value is the min of estimated partition size.

Reply:
Done